.zenodo.json

```diff
@@ -73,6 +82,11 @@
       "affiliation": "TU Dresden",
       "type": "Other"
     },
+    {
+      "name": "Hübl, Axel",
+      "affiliation": "Lawrence Berkeley National Laboratory",
+      "orcid": "0000-0003-1943-7141"
+    },
```

Comment:
The type others is missing and is required any name in contributors: `"type": "Other"`

Reply:
Done